src/components/MovieInfo.tsx

```diff
@@ -15,25 +15,31 @@ export type DataChildren = {
 };
 
 const MovieInfo: React.FC<MovieInfoProps> = ({ data }) => {
-
   return (
     <div className="container">
       <div className="row justify-content-between align-items-center">
```

Comment:
Please update the following section to the suggested code below:
`<div className="row justify-content-between">`